project/plugins.sbt

```diff
@@ -11,3 +11,5 @@ addSbtPlugin("com.dwijnand" % "sbt-dynver" % "4.1.1")
 addSbtPlugin("org.scalameta" % "sbt-mdoc" % "2.3.3")
 
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "1.1.1")
+
+libraryDependencies += "ai.kien" %% "python-native-libs" % "0.2.4"
```

Comment:
Hmm, won't this just fetch from Maven Central? It seems we have a tricky recursive dependency with this. I guess perhaps we could always use the previous version, but that isn't ideal.